### ISSUE TITLE
Figure out Wasp's deployment story on AWS

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#1828]
Right now we have instructions for deploying Wasp to Fly, Railway, Heroku.

Of course, Wasp can right now be deployed to AWS already, since we give you Docker for server and static files for frontend, but we don't have nice instructions and recommendations on how to best go about it.

Advice we got from AWS pplz:
> Amazon Lightsail would be the easier and cheaper solution for users that are not experienced with AWS. This would also limit the ‘blast-radius’ in terms of what they are able to provision as it is on a different console than the full AWS Console.
> You could have Amazon Lightsail for ‘non-experienced users’ and AWS App Runner for users with experience in a cloud environment.

**Comment by Martinsos** [comment]
@infomiho also mentioned sst.dev and Pulumi as interesting options.

**Comment by rugs07** [comment]
Did you have instructions to deploy it on AWS?

**Comment by Martinsos** [comment]
Nope, this is waht this issue is about! Would be great to have some.
